src/main/java/de/maxhenkel/audioplayer/command/AudioPlayerCommands.java

```diff
@@ -399,15 +400,15 @@ private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(Liter
                                     ItemStack itemInHand = player.getItemInHand(InteractionHand.MAIN_HAND);
                                     String customName = StringArgumentType.getString(context, "custom_name");
                                     if (validator.test(itemInHand)) {
-                                        processShulker(context, itemInHand, sound, customName);
+                                        processShulker(context, itemInHand, itemTypeName, sound, customName);
                                     } else {
                                         context.getSource().sendFailure(Component.literal("You don't have a %s in your main hand".formatted(itemTypeName)));
                                     }
                                     return 1;
                                 })));
     }
 
-    private static void processShulker(CommandContext<CommandSourceStack> context, ItemStack itemInHand, UUID soundID, @Nullable String name) {
+    private static void processShulker(CommandContext<CommandSourceStack> context, ItemStack itemInHand, String itemTypeName, UUID soundID, @Nullable String name) {
         ListTag shulkerContents = itemInHand.getTagElement("BlockEntityTag").getList("Items", 10);
```

Comment:
Use `BlockItem.BLOCK_ENTITY_TAG`

Reply:
Fixed, also set "Items" to `ShulkerBoxBlockEntity.ITEMS_TAG` instead